tests/integration/ar-io-writable.test.ts

```diff
@@ -49,7 +50,7 @@ describe('ArIOWriteable', () => {
   it('should successfully increase delegate stake', async () => {
     const tx = await arIO.increaseDelegateStake({
       target: gatewayAddress,
-      qty: 101,
+      qty: new IOToken(100).toMIO(),
```

Comment:
would be nice if the test checked the before/after value on the gateway matches expected values after increase/decreaseDelegateStake

Reply:
[45e3521](https://github.com/ar-io/ar-io-sdk/pull/92/commits/45e3521f7701fff1be0870dcf2f8af65e68d1cf8)